airbyte-integrations/connectors/source-chargebee/integration_tests/invalid_config.json

```
@@ -1,3 +1,5 @@
 {
-  "todo-wrong-field": "this should be an incomplete config file, used in standard tests"
+  "site": "airbyte-test",
+  "site_api_key": "<your_api_key>",
```

```suggestion
  "site_api_key": "fake-api-key-1",
```
That it was not identical to `sample_config.json` and corresponded to the title. WDYT?

---

Done.